admin3.php

```diff
@@ -34,27 +34,25 @@
 			<!-- login box -->
 			<div class = "login_box">
 
-				<b><p style = "color: black; font-size: 2vw; margin: 5.5vw 0vw 0vw 3.7vw;">Update Username</p></b>
+				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Update Password</p></b>
 
 				<!-- Form for input fields -->
 				<h3><form method = "post" id= "01_login">
 
-					<p style = "margin: -2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Existing Username:</p>
-					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "ExistingUserName" placeholder="Your Existing Username..."/>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 4vw 0vw 0vw 4.2vw;" type = "text" name = "username" placeholder="Existing Username..."/>

```

Comment:
Below is where I have adjusted the margins and other style elements for the input fields.